nixos/modules/system/boot/loader/limine/limine-install.py

```diff
@@ -0,0 +1,303 @@
+#!@python3@/bin/python3 -B
```

Comment:
I'm going to be taking a look, but it will take a while to go through this.

Would you be opposed to adding typing hints?

Reply:
No, I'll get to it as soon as I can.